FastStyleTransfer/Demo/Helpers.swift

```diff
@@ -0,0 +1,92 @@
+import Foundation
+import TensorFlow
+import FastStyleTransfer
+
+// Make model importable
+extension TransformerNet: ImportableLayer {}
+
+func parseArgs<T>(into obj: inout T, with params: [String: WritableKeyPath<T, String?>]) {
```

Comment:
Avoid abbreviations.
```suggestion
func parseArguments<T>(into obj: inout T, with params: [String: WritableKeyPath<T, String?>]) {
```

Reply:
Done